assets/components/atoms/collapse/collapse.scss

```diff
@@ -0,0 +1,75 @@
+@charset 'utf-8';
+
+@mixin collapse-btn() {
+  $icon-size: 1rem;
+  $icon-size-half: $icon-size/2;
+  $icon-thickness: 2px;
+  $icon-thickness-half: $icon-thickness/2;
+
+  display: block;
+  position: relative;
+  width: 100%;
+  margin-bottom: 0;
+  padding: 0.9rem 0;
+  border-top: 1px solid $gray-300;
+  font-size: 1.1rem;
+  text-align: left;
+  font-weight: 800;
+
+  &:hover { cursor: pointer; }
+
+  &[aria-expanded="true"] {
+    &:before { transform: scaleY(0); }
+    &:after { background: $red; }
+  }
+
+  &:after {
+    content: ' ';
+    display: block;
+    position: absolute;
+    top: calc(50% - #{$icon-thickness-half});
+    right: 0;
+    width: $icon-size;
+    height: $icon-thickness;
+    background: $gray-400;
+    transition: background 0.2s;
+  }
+
+  &:before {
+    content: ' ';
+    display: block;
+    position: absolute;
+    top: calc(50% - #{$icon-size-half});
+    right: calc(#{$icon-size-half} - #{$icon-thickness-half});
+    width: $icon-thickness;
+    height: $icon-size;
+    background: $gray-400;
+    transition: transform 0.3s;
+  }
+}
+
+.collapse-title {
```

Comment:
The focus is very accessible but is a bit sad design-wise. Could we style it soberly? (like a grey background or such, to avoid the big blue outline)